bin/docker-assets-precompile

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
```

Comment:
@rosle Since this is a ruby script, should the file name end in `.rb` ?

Reply:
I think we can go without the `.rb` @ankitkalia1195 as it won't make the confusion with the Ruby class

Also Rails don't use `.rb` in their `ruby script` as well, for example the `bin/setup` file

![image](https://user-images.githubusercontent.com/11751745/93874150-22439000-fcfd-11ea-8b6c-ffbf69c1c732.png)

Reply:
@andyduong1920 Sure, got it